src/components/accession2/withdraw/WithdrawModal.tsx

```diff
@@ -464,7 +464,7 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
                 <Box>{strings.CT}</Box>
               ) : (
                 <Dropdown
-                  options={weightUnitsV2()}
+                  options={weightUnitsV2(userPreferences.preferredWeightSystem as string)}
```

Comment:
suggestion, create another function in units.ts which returns weightUnitsV2 based off user preference.. we can avoid some repetition code here